core/server/master/src/main/java/alluxio/master/metrics/MetricsStore.java

```diff
@@ -44,6 +45,10 @@
 @ThreadSafe
 public class MetricsStore {
   private static final Logger LOG = LoggerFactory.getLogger(MetricsStore.class);
+  // The following fields are added for reducing the string processing
+  // for MetricKey.WORKER_BYTES_READ_UFS and MetricKey.WORKER_BYTES_WRITTEN_UFS
+  private static final String BYTES_READ_UFS = "BytesReadPerUfs";
```

Comment:
These constants are available in `WorkerMetrics`?

Reply:
I removed all the `WorkerMetrics`, `MasterMetrics`, `ClientMetrics` to the metric key which define all metrics with full name `Worker.BytesReadPerUfs`.